Dockerfile

```diff
@@ -1,19 +1,19 @@
 # [Node Stage to get node_modolues and node dependencies]
-FROM node:8.16.0-buster-slim as node_stage
+FROM node:8.16.0-buster-slim as node_base
+# [Python Stage for Django web server]
+FROM python:3.10.14-slim-bullseye as python_base
```

Comment:
I've tried but when I run `ldd /.venv/bin/uwsgi`  in the runner stage, `uwsgi`'s shared libraries is not installed properly. 

```
uwsgi: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
```

Speaking of reducing the final image size, is it necessary everything in the prod stage?
```
COPY --chown=docker:nogroup . $APP_DIR/
```

Maybe we could change it to
```
COPY --chown=docker:nogroup ./src /usr/local/app/src
COPY --chown=docker:nogroup ./logs /usr/local/app//logs
COPY --chown=docker:nogroup ./__init__.py /usr/local/app/__init__.py
```

What do you think?

Reply:
how can i reproduce this? is there any experimental branch or minimum code for it?

Reply:
uwsgi is in maintenance mode. it could be an issue too.

Reply:
I've pushed it to `feature/add-runner-stage`!

Reply:
unresolved comment
I'll get back to this when I'm available

Reply:
I was wondering if it's ok to explore adding runner stage in the next PR? 
I feel it would be better to give current Dockerfile some time testing before dividing it into more stages.
What do you think?

Reply:
ticket opened: https://github.com/orgs/pycontw/projects/6/views/1?pane=issue&itemId=62256808